integration-tests/jdbc/src/main/java/org/apache/camel/quarkus/component/jdbc/CamelResource.java

```diff
@@ -88,22 +87,22 @@ public String getSpeciesById(@PathParam("id") String id) throws Exception {
     @Produces(MediaType.TEXT_PLAIN)
     public String getSpeciesByIdWithSelectList(@PathParam("id") String id) throws Exception {
         List<LinkedHashMap<String, Object>> result = template
-                .requestBody("jdbc:camel-ds?outputType=SelectList", "select * from camels where id = " + id, List.class);
+                .requestBody("jdbc:cameldb?outputType=SelectList", "select * from camels where id = " + id, List.class);
 
         if (result.isEmpty()) {
             throw new IllegalStateException("Expected at least 1 camel result but none were found");
         }
 
         LinkedHashMap<String, Object> data = result.get(0);
-        return data.get("SPECIES") + " " + data.get("ID");
+        return data.toString();
```

Comment:
Is it really better to use toString() ? What happen if the toString() implementation changes ?